docs/readthedocs/source/doc/LLM/Overview/install_gpu.md

```diff
@@ -162,6 +252,10 @@ We recommend using [miniconda](https://docs.conda.io/en/latest/miniconda.html) t
    ``bigdl-llm`` is tested with Python 3.9, 3.10 and 3.11. Python 3.9 is recommended for best practices.
 ```
 
+.. important::
```

Comment:
The style is wrong
![image](https://github.com/intel-analytics/BigDL/assets/21994046/e7ec8ce9-9e96-481c-aae3-03bd967fdb44)